src/wwwroot/partials/downloads/downloads.html

```diff
@@ -19,8 +19,8 @@
         </tr>
         <tr class="column-records" ng-repeat="r in records">
           <td>{{r.name}}</td>
-          <td>{{r.start_date | date:'MMM':'+0' | translate}} {{r.start_date | date:'dd, yyyy':'+0'}}</td>
-          <td>{{r.end_date | date:'MMM':'+0' | translate}} {{r.end_date | date:'dd, yyyy':'+0'}}</td>
+          <td>{{r.start_date | date: 'mediumDate'}}</td>
+          <td>{{r.end_date | date: 'mediumDate'}}</td>
```

Comment:
Did you verify if it is dealing fine with different browser timezones? (By the moment we should show UTC dates)